src/com/synopsys/integration/pipeline/versioning/GithubReleaseStage.groovy

```diff
@@ -1,172 +1,92 @@
-        this.artifactDirectory = null
-        this.branch = branch
-    }
+import java.text.SimpleDateFormat
+
+class GithubReleaseStage extends Stage{
+    public static String RELEASE_FILE = 'release.json'
+    private String releaseOwner
+    private String releaseRepo
+    private String releaseTagName
+    private String releaseTargetCommitish
+    private String releaseName
+    private String releaseBody
+    private String githubCredentialsId
+    private String targetCommitish
 
-    GithubReleaseStage(PipelineConfiguration pipelineConfiguration, String stageName, boolean runRelease, String artifactPattern,
-                       String artifactDirectory, String branch) {
+    GithubReleaseStage(PipelineConfiguration pipelineConfiguration, String stageName, String releaseOwner, String releaseRepo, String githubCredentialsId) {
         super(pipelineConfiguration, stageName)
-        this.runRelease = runRelease
-        this.artifactFile = null
-        this.artifactPattern = artifactPattern
-        this.artifactDirectory = artifactDirectory
-        this.branch = branch
+        this.releaseOwner = releaseOwner
+        this.releaseRepo = releaseRepo
+        this.githubCredentialsId = githubCredentialsId
     }
 
     @Override
     void stageExecution() throws PipelineException, Exception {
-        if (!runRelease) {
-            getPipelineConfiguration().getLogger().info("Skipping the ${this.getClass().getSimpleName()} because this is not a release.")
-            return
-        }
-        String version = getPipelineConfiguration().getScriptWrapper().getJenkinsProperty(GITHUB_RELEASE_VERSION)
-        if (StringUtils.isBlank(version)) {
-            throw new PrepareForReleaseException("Could not find the \"${GITHUB_RELEASE_VERSION}\" environment variable. Will not perform the GitHub release.")
-        }
-        List<String> options = []
-        options.add('-o')
-        options.add(owner)
-        options.add('-v')
-        options.add(version)
-        if (StringUtils.isNotBlank(artifactFile)) {
-            options.add('-f')
-            options.add(artifactFile)
-        }
-        if (StringUtils.isNotBlank(artifactPattern)) {
-            options.add('-t')
-            options.add(artifactPattern)
-        }
-        if (StringUtils.isNotBlank(artifactDirectory)) {
-            options.add('-d')
-            options.add(artifactDirectory)
-        }
-        if (StringUtils.isNotBlank(project)) {
-            options.add('-p')
-            options.add(project)
-        }
-        options.add('-m')
-        options.add("\"${releaseDescription}\"")
-
-        GithubBranchParser githubBranchParser = new GithubBranchParser()
-        GithubBranchModel githubBranchModel = githubBranchParser.parseBranch(branch)
-
-        options.add('-br')
-        options.add(githubBranchModel.getBranchName())
+        try {
+            String timeStamp = new SimpleDateFormat("yyyy.MM.dd.HH.mm.ss").format(new Date())
```

Comment:
Can you put a space in between the date and the time?

Reply:
Yes, I have updated the date format to be more clear, for example now it will say "Aug 1, 2022 12:53:48"